rbc/tests/test_omnisci_column_arguments.py

```diff
@@ -81,7 +81,7 @@ def text_rbc_copy_cccc_rowmul(x, x2, x3, x4, m, y, y2, y3, y4):
                           'cursor(i4,i8);f8', 'cursor(i4,i8);cursor(f8,f4)',
                           'cursor(i4,i8,f8,f4)'])
 def test_copy(omnisci, inputs):
-    omnisci.require_version((5, 6), 'Requires omniscidb-internal PR 5134')
+    omnisci.require_version((5, 7), 'Requires omniscidb-internal PR 5134')
```

Comment:
This is to avoid pytest to execute this test?  If so, perhaps is it better to skip the test with `@pytest.mark.skip(reason)` decorator and remove it after a new version of omniscidb is released.

Reply:
https://github.com/omnisci/omniscidb-internal/pull/5134 has been merged. So, this test can be enabled again.